scripts/stack/configure-stack.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o pipefail
+
+if [[ ${CURDIR:-} == "" ]]; then
+    export CURDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+fi
+source $CURDIR/read-env.sh $CURDIR/.env
+
+PYTHON_EXECUTABLE=""
+
+if which python3 > /dev/null; then
+    PY_VERSION=`python3 --version`
+    PYTHON_EXECUTABLE="python3"
+elif which python > /dev/null; then
+    PY_VERSION=`python --version`
+    PYTHON_EXECUTABLE="python"
+fi
+
+pushd $PROJECT_ROOT
+    if [[ "${CONFIG_FILE:-}" == "" ]]; then
+        CONFIG_FILE="${PROJECT_ROOT}/scripts/stack/connectors-config/config.yml"
+    fi
+
+    CONNECTORS_EXE="${PROJECT_ROOT}/bin/connectors"
+    if [ ! -f "$CONNECTORS_EXE" ]; then
+        echo "Could not find a connectors executable, running 'make clean install'"
+
+        if [ $PYTHON_EXECUTABLE == "" ]; then
+            echo "Could not find a suitable Python 3 executable..."
+            exit 2
+        fi
```

Comment:
Should this happen before everything else?

Reply:
This is only needed in the case of it needing to run `make clean install`... so, it's not needed overall.